PLAT-11372: Support new UX 1.1 with garnet/WheelSliderPanel

SkylerBaek opened the pull request:
Support new UX 1.1 with garnet/WheelSliderPanel

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

ybsung committed:
Added WheelSliderPanel into FormSample

Enyo-DCO-1.1-Signed-off-by: YB Sung <yb.sung@lge.com>

SkylerBaek committed:
PLAT-11372: Support new UX 1.1 with garnet/WheelSliderPanel

WheelSliderController is a panel, not a just controller. So samples are updated to not to wrap a controller with another panels.
FormSample is now working with WheelSliderControllerSample.

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

ybsung commented on garnet/src/WheelSliderControllerSample.js:
```diff
@@ -45,12 +32,6 @@ var WheelSliderControllerPanel = kind({
-	tapOK: function(inSender, inEvent) {
-		this.doResult({msg: 'OK button tapped !!'});
-	},
 	changingEventHandler: function(inSender, inEvent) {
```
It's better to define handlers for onChange, onChanging

```
    handlers: {
        onChange: 'changeEventHandler',
        onChanging: 'changingEventHandler'
    },
```

SkylerBaek replied:
@ybsung, As we discussed, all event handlers for onChange and onChanging will be removed and sample code will be written based on valueChanged().

ybsung commented on garnet/src/WheelSliderControllerSample.js:
```diff
@@ -2,40 +2,27 @@ require('garnet');
+	minimumValue: 0,
+	maximumValue: 100,
+	stepValue: 5,
+	value: 50,
```
It's better to define minimumValue, maximumValue, stepValue and value in not WheeSliderPanel but g.sample.WheelSliderController

SkylerBaek replied:
WheelSliderController always needs initial values for all properties above to work properly. So we need to guide developers to use WheelSliderController instance with initial values. Let's keep this as is.

SkylerBaek committed:
PLAT-11372: Support new UX 1.1 with garnet/WheelSliderPanel

WheelSliderController is renamed to WheelSliderPanel

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

SkylerBaek commented:
@ybsung, I updated this PR as we discussed. Please review this.

SkylerBaek committed:
PLAT-11372: Support new UX 1.1 with garnet/WheelSliderPanel

Bug fixed: a slider was not rendered due to a wrong value.

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

SkylerBaek commented:
@ybsung, a known issue is fixed.

SkylerBaek committed:
Merge remote-tracking branch 'refs/remotes/origin/PLAT-11357-Garnet-NewUX' into PLAT-11372-sc.baek

SkylerBaek committed:
Merge remote-tracking branch 'refs/remotes/origin/PLAT-11357-Garnet-NewUX' into PLAT-11372-sc.baek

Conflicts:
	garnet/src/FormSample.js
Resolved.

Enyo-DCO-1.1-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)
